tools/coverity/models.c

```diff
@@ -0,0 +1,3 @@
+void uvisor_noreturn(void) {
```

Comment:
This needs our copyright header and license at the top.

Reply:
of course..

Reply:
Should start copyright from the year the file was born (only 2017).